The build error seems legitimate:

```console
-- The C compiler identification is Clang 21.0.0
-- The CXX compiler identification is Clang 21.0.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /android-ndk-r29/toolchains/llvm/prebuilt/linux-x86_64/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /android-ndk-r29/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- HEXL BRANCH:                   
-- CMAKE_BUILD_TYPE:              Debug
-- CMAKE_C_COMPILER:              /android-ndk-r29/toolchains/llvm/prebuilt/linux-x86_64/bin/clang
-- CMAKE_CXX_COMPILER:            /android-ndk-r29/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++
-- HEXL_BENCHMARK:                OFF
-- HEXL_COVERAGE:                 OFF
-- HEXL_DEBUG:                    ON
-- HEXL_DOCS:                     OFF
-- HEXL_EXPERIMENTAL:             OFF
-- HEXL_SHARED_LIB:               0
-- HEXL_TESTING:                  OFF
-- HEXL_TREAT_WARNING_AS_ERROR:   OFF
-- HEXL_FPGA_COMPATIBILITY:       0
-- CMAKE_CXX_FLAGS_DEBUG -fno-limit-debug-info   
-- CMAKE_CXX_FLAGS_RELEASE -O3 -DNDEBUG   
-- CMAKE_CXX_FLAGS_MINSIZEREL -Os -DNDEBUG
-- CMAKE_CXX_FLAGS_RELWITHDEBINFO -O2 -g -DNDEBUG
-- CMAKE_CXX_FLAGS -g -DANDROID -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security -frtti -fexceptions  -fPIC  
-- Compile flag not found: HEXL_HAS_AVX512DQ
-- Compile flag not found: HEXL_HAS_AVX512IFMA
-- Compile flag not found: HEXL_HAS_AVX512VBMI2
-- Compile flag not found: HEXL_HAS_AVX256
-- CMAKE_INSTALL_LIBDIR:     lib
-- CMAKE_INSTALL_INCLUDEDIR: include
-- CMAKE_INSTALL_PREFIX:     /mnt/vcpkg-ci/p/hexl_x64-android/debug
-- CpuFeatures: found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Check if compiler accepts -pthread
-- Check if compiler accepts -pthread - yes
-- Found Threads: TRUE
-- Found EASYLOGGINGPP: /mnt/vcpkg-ci/installed/x64-android/debug/lib/libeasyloggingpp.a
-- easyloggingpp library found
-- easyloggingpp: pre-installed easyloggingpp found
-- Configuring incomplete, errors occurred!
```

```
fatal: not a git repository (or any of the parent directories): .git
CMake Error: try_run() invoked in cross-compiling mode, please set the following cache variables appropriately:
   CAN_RUN (advanced)
   CAN_RUN__TRYRUN_OUTPUT (advanced)
For details see /mnt/vcpkg-ci/b/hexl/x64-android-dbg/TryRunResults.cmake
```

--- a/ports/hexl/vcpkg.json
+++ b/ports/hexl/vcpkg.json
@@ -4,7 +4,7 @@
   "description": "Intel® HEXL is an open-source library which provides efficient implementations of integer arithmetic on Galois fields.",
   "homepage": "https://github.com/IntelLabs/hexl",
   "license": "Apache-2.0",
-  "supports": "x64",
+  "supports": "x64 & !android",
   "dependencies": [
     "cpu-features",
     "easyloggingpp",

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -256,7 +256,6 @@ gz-tools:arm-neon-android=fail
 gz-tools:arm64-android=fail
 gz-tools:x64-android=fail
 halide:x64-windows-static=fail
-hexl:x64-android=fail
 hwloc:arm-neon-android=fail
 hwloc:arm64-android=fail
 hwloc:x64-android=fail

--- a/versions/h-/hexl.json
+++ b/versions/h-/hexl.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "f9b90aa4040fe88b72df0996ee04adba8e81aa60",
+      "git-tree": "1a47601a308ef41c32571442efedb97c61bb3db3",
       "version": "1.2.6",
       "port-version": 0
     },

